src/main.py

```diff
@@ -90,9 +90,10 @@ def _create_new_version_artifacts(args):
 
 
 def _copy_static_files(base_version_dir, new_version_dir):
-    for name in ['gpu.arg_based_env.in', 'Dockerfile']:
-        for f in glob.glob(f'{base_version_dir}/{name}'):
-            shutil.copy2(f, new_version_dir)
+    for f in glob.glob(f'{base_version_dir}/gpu.arg_based_env.in'):
+        shutil.copy2(f, new_version_dir)
+    for f in glob.glob(os.path.relpath(f'template/Dockerfile')):
```

Comment:
I am wondering if we can get rid of `Dockerfile` completely from the `build_artifacts` directory going forward?

Like, we can have a single `Dockerfile` in the `template` directory and use it as is all the time.

I guess one problem is that the `build_artifacts` will stop containing the `Dockerfile` which was used to create a specific version. However, what's the purpose of that anyway today? It's not as if having that (as we do today) helps with reproducibility or anything.

Reply:
Never mind, I see you addressed this point already in the PR description. I am fine with how you are thinking.

Reply:
Just for accountability , we need that in the Build artifacts folder .